src/controllers/csi/config.go

```diff
@@ -37,8 +37,8 @@ const (
 
 var MetadataAccessPath = filepath.Join(DataPath, "csi.db")
 
-type CSIOptions struct {
-	NodeID   string
+type CsiOptions struct {
```

Comment:
this rename make stuff a bit inconstant

we have `CSIDriverServer ` and `CSIGarbageCollector`

Reply:
Reverted this change because k8s core uses same (wrong:slightly_smiling_face:) nanimg convention (e.g. `v1.CSIVolumeSource`)

Reply:
Reverted this change because k8s core uses same (wrong:slightly_smiling_face:) nanimg convention (e.g. `v1.CSIVolumeSource`)